server/src/main/java/org/elasticsearch/index/mapper/MapperService.java

```diff
@@ -451,15 +421,11 @@ static void validateTypeName(String type) {
             if (updatedDocumentMapper != newMapper) {
                 // update both mappers and result
```

Comment:
nit: comment can be adapted since result is not there now

Reply:
++